contracts/Optimism_SpokePool.sol

```diff
@@ -12,22 +13,24 @@ import "./SpokePoolInterface.sol";
  */
 
 contract Optimism_SpokePool is CrossDomainEnabled, SpokePoolInterface, SpokePool {
-    // Address of the L1 contract that acts as the owner of this SpokePool.
-    address public override crossDomainAdmin;
-
-    event SetXDomainAdmin(address indexed newAdmin);
+    event TokensBridged(
```

Comment:
Do we need this event to be different per chain? Or could we standardize and put in the base class?

Reply:
I think it should be diff per chain since each chain will use different params to bridge tokens. For example, Arbitrum uses something called a `maxSubmissionCost` whereas optimsim does not

Reply:
Gotcha, okay. Could make sense to have a standard event that we use for all contracts, if the specific parameters aren't relevant. Just something to think about later. Not something I think we should change now.

Reply:
Hmmm ill add a TODO